RNTester/js/utils/RNTesterList.macos.js

```diff
@@ -11,6 +11,46 @@
 // TODO(macOS ISS#2323203)
 
 /* $FlowFixMe allow macOS to share iOS file */
-const RNTesterList = require('./RNTesterList.ios.js');
+const RNTesterListIOS = require('./RNTesterList.ios.js');
+
+const APIExamples = [
+  ...RNTesterListIOS.APIExamples.filter(
+    ex =>
```

Comment:
Shouldn't need status bar for macOS either

Reply:
Or AppleTV, Checkbox, and a couple others. There are actually quite a few UnimplementedViews for macOS that we can just not include right now until we have support for them. If you search for `UnimplementedViews` you can see all the components that don't actually exist on macOS and just have dummy implementations we can probably just omit here. Once we add implementations, part of the workflow can be to add it to the test list.

Reply:
Feel free to adjust the test list after the merge! 🙂